Author: ?
Description: breakage in python2.5 or pygtk
 Temporary workaround for what seems to be breakage in python2.5 or
 pygtk (see #481480, #481837, #481977), by closing stdin explicitly
--- 9-ccsm.orig/ccsm
+++ 9-ccsm/ccsm
@@ -132,4 +132,6 @@ if dbusObj is not None:
 idle = ccm.IdleSettingsParser(context, mainWin)
 mainWin.show_all()
 
+import os
+os.close(0)
 gtk.main()
